src/floki/core.cljs

```diff
@@ -61,7 +68,7 @@
 
 (defn stdin-handler
   [buf]
-  (rf/dispatch [:input/set (convert buf)]))
+  (rf/dispatch [:input/set (-> buf convert)]))
```

Comment:
Why?

Reply:
no particular reason - it's the same as the other handler like this, which is appealing. was playing with more steps here and it got left behind, I think.